package.json

```diff
@@ -1,24 +1,25 @@
 {
   "name": "business-create-ui",
-  "version": "5.3.1",
+  "version": "5.4.0",
   "private": true,
   "appName": "Create UI",
   "sbcName": "SBC Common Components",
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
-    "lint:nofix": "vue-cli-service lint --no-fix",
-    "test:unit": "vue-cli-service test:unit --testPathPattern --coverage",
-    "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "lint": "eslint . --ext .js,.ts,.vue",
+    "test:unit": "vitest run",
+    "test:coverage": "vitest run --coverage"
   },
   "dependencies": {
     "@babel/compat-data": "^7.21.5",
     "@bcrs-shared-components/approval-type": "1.0.19",
+    "@bcrs-shared-components/base-address": "2.0.1",
```

Comment:
`BaseAddress` was previously being used from "sbc-common-components". Now, the component is imported from the "bcrs-shared-components" repo. I've updated all the corresponding imports accordingly too in the code.

Reply:
Thank you for this

Reply:
I know there was a (test?) issue with using the shared BaseAddress component and Karim reverted that change, so it's now out of scope for this PR.

However, Travis, did you manage to use the shared BasedAddress component in an app? And unit test it? Or is there maybe something still wrong/needed with it?